src/util/octal_utils.e

```diff
@@ -16,42 +16,72 @@ class
 
 feature -- Format check
 
-	is_octal_natural_16_string (a_string: READABLE_STRING_8): BOOLEAN
-			-- Indicates whether `a_string' represents an octal encoded NATURAL_16
+	is_octal_formatted (a_string: READABLE_STRING_8): BOOLEAN
```

Comment:
I guess this code is a bit better than checking is_number_sequence and checking that no 8 or 9 are included...